app/ui/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -619,6 +611,7 @@ public void openFolder(String folderName) {
 
     public void openUnifiedInbox() {
         drawer.selectUnifiedInbox();
+        drawer.setUserAccounts(preferences.getAccounts());
```

Comment:
This should probably use AccountsLiveData in some manner, which ties in with #3597

Reply:
Yes, I'll keep an eye on that.